idaes/core/util/testing.py

```diff
@@ -79,30 +80,31 @@ def initialization_tester(m, dof=0, **init_kwargs):
     Raises:
         AssertionErrors is an issue is found
     """
+    unit = getattr(m.fs, unit_name)
```

Comment:
Whilst this allows a custom unit name, it still assumes `m.fs.` i the flowsheet object. If we are going to allow this flexibility, then maybe we should go all the way and make "unit" a required input with the actual model object.

Reply:
Sure.  That's no problem.  Will change.